lib/puppet/face/node/clean.rb

```diff
@@ -1,19 +1,18 @@
 Puppet::Face.define(:node, '0.0.1') do
   action(:clean) do
 
-    summary _("Clean up signed certs, cached facts, node objects, and reports for a node stored by the puppetmaster")
```

Comment:
This is wrong now. We updated this face to attempt to load the puppetserver-ca gem, see below.